x-pack/test/cases_api_integration/security_and_spaces/tests/common/cases/get_metrics.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+import { deleteAllCaseItems, getCaseMetrics } from '../../../../common/lib/utils';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const es = getService('es');
+  const kibanaServer = getService('kibanaServer');
+
+  describe('metrics', () => {
+    const closedCaseId = 'e49ad6e0-cf9d-11eb-a603-13e7747d215z';
+
+    before(async () => {
+      await kibanaServer.importExport.load(
```

Comment:
Why we need to load cases?

Reply:
So this is code I wrote in the backend. This file will actually be deleted because I moved it to a different directory (https://github.com/elastic/kibana/tree/main/x-pack/test/cases_api_integration/security_and_spaces/tests/common/metrics).

To answer you question though, we don't really need to, we could just as easily do the create case and then close case so we have the open and close timestamps. 

I figured it'd be easier to have the open and close dates hardcoded and deterministic:

```
            creationDate: '2021-06-17T18:57:41.682Z',
            closeDate: '2021-06-17T18:57:42.682Z',
```

This was also a bit of an experiment on my part to see if we could get the importing cases to work using the kbnArchiver instead of using esArchiver. We have to specifically move our esArchiver calls to occur after all our tests otherwise when we unload the `.kibana` index it messes up the index and breaks any tests that leverage spaces and users that occur after that:

https://github.com/elastic/kibana/blob/main/x-pack/test/cases_api_integration/security_and_spaces/tests/common/index.ts#L44

By using the `kibanaServer` we no longer run into that problem 😃  

More details in this issue: https://github.com/elastic/kibana/issues/102552

Reply:
I see. Thanks for the explanation!